docs/tutorials/speech_recognition/ctc.md

```diff
@@ -1,5 +1,11 @@
 # Connectionist Temporal Classification
 
+```python
+
+import mxnet as mx
+print(mx.__version__)
```

Comment:
Should remove as don't have this anywhere else, and it's result isn't shown either.